grails-app/services/au/org/ala/ecodata/ParatooService.groovy

```diff
@@ -311,6 +320,18 @@ class ParatooService {
                 log.warn "Error saving form with id: "+id+", name: "+name
             }
         }
+
+        List allProtocolForms = ActivityForm.findAll {
+            externalIds {
+                idType == ExternalId.IdType.MONITOR_PROTOCOL_GUID
+            }
+            status != Status.DELETED
+        }
+
+        List deletions = allProtocolForms.findAll{it.externalIds.find{it.idType == ExternalId.IdType.MONITOR_PROTOCOL_GUID && !(it.externalId in guids)}}
+        deletions.each { ActivityForm activityForm ->
```

Comment:
Is it intentional not to delete or update the activity form?

Reply:
Yes, in case the form has been used I think it's unsafe to automatically delete it.  I'm just logging for now so we can manually review any deleted protocols.  In theory this won't happen once we go into production.